<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grbox_Comport_Control">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>261</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Comport Control</string>
    </property>
    <widget class="QLabel" name="lbl_Port_Name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Baudrate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Data_Bits">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Bits</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Stop_Bits">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop bits</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Parity_Bits">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity Bits</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combox_Port_Name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combox_Baudrate">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comBox_Data_Bits">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combox_Stop_Bits">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combox_Parity_bits">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbox_Display_Data">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>351</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Display ƒêata</string>
    </property>
    <widget class="QPushButton" name="btnClearData_Display">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>500</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear data</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_data_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>341</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbox_Comport_Status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Comport Status</string>
    </property>
    <widget class="QPushButton" name="btnconnect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Close">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Status_Comport">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Send Data</string>
    </property>
    <widget class="QPushButton" name="btnClear_Send_Data">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSend_Data">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Data</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtEdit_Send_Data">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grBox_Chart">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>541</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Graph</string>
    </property>
    <widget class="QCustomPlot" name="Plot_chart" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>501</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearData_plot">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>470</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear data</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1268</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
